app/views/comments/index.html.erb

```diff
@@ -0,0 +1 @@
+<h2>Hello there!</h2>
```

Comment:
Probably don't need an index view for comments, just because you wouldn't want to view them all together as a list, but without their posts. If you were viewing them along with the posts they belong to, it would essentially be the same as a posts index view.